cms/static/sass/views/_settings.scss

```diff
@@ -838,6 +838,184 @@
       }
     }
 
+    .add-new-data {
+      background-color: $green;
+      border-color: $green;
+      color: $white;
+      display: inline-block;
+      padding: 10px 20px;
+      border-radius: 3px;
+      margin-bottom: 10px;
+      &:hover{
+        background-color: $green-l1;
+        border-color: $green-l1;
+        box-shadow: 0 2px 1px rgba(0,0,0,0.2);
+      }
+      &.add-course-learning-info {
+        margin-top: 10px;
+      }
+    }
+
+    &.course-learning-info {
+      .list-input {
+        .course-settings-learning-fields {
+          .field {
+            .input-learning-info {
+              width: 88%;
+              display: inline-block;
+            }
+            .input-learning-info-stretch {
+              width: 100%;
+              display: inline-block;
+            }
+            input[type="button"] {
+              width: 10%;
+              margin-left: 1%;
+              font-size: 14px;
```

Comment:
We generally do not set explicit font sizes, and use variables. Is this CSS consistent with how similar things are styled on this page?